recipes/fasta_attributes_asset_recipe.yaml

```diff
@@ -0,0 +1,28 @@
+name: fasta_attributes
+version: 0.0.1
+output_asset_class: fasta_attributes
+description:
+  Attributes to Sequences in the FASTA file. 
+  mito_name (Mitocondrial chromosome name) and macs_gsize (Genome size).
+inputs:
+  files: {}
+  params:
+    name:
+      description: Mitocondrial chromosome name
+    size:
+      description: Genome size
+  assets: {}
+container: docker.io/databio/refgenie
+default_tag: ''
+command_template_list:
```

Comment:
The commands here should create the asset of interest. These commands aren't going to create any persistent output